Code Analysis.md

```diff
@@ -110,12 +110,11 @@ According to the [CWE-916](https://cwe.mitre.org/data/definitions/916.html), the
 ### B312	Telnet-related functions are being called. Telnet is considered insecure. Use SSH or some other encrypted protocol.
 Telnet is a communication protocol running on port 23 that transmits and receives data in plain text. [CWE-311](https://cwe.mitre.org/data/definitions/311.html) explains that the weakness is due to the software not encrypting sensitive or critical data. This gives actors the ability to eavesdrop. The file [sensor.py](https://github.com/home-assistant/core/blob/dev/homeassistant/components/hddtemp/sensor.py) uses telnet to run the command HDDTemp as a daemon to gain temperature information of the disk specified in the JSON file (line 120). Considering the vast array of devices and possible configurations, Home Assistant transmits this non-sensitive data through telnet. Although risky, the Home Assistant team's assumption that the local network is safe and the data transmitted is non-sensitive, we decided not to submit a pull request.
 
-### B321	FTP-related functions are being called. FTP is considered insecure. Use SSH/SFTP/SCP or some other encrypted protocol.
-
 ### B401	A telnet-related module is being imported.  Telnet is considered insecure. Use SSH or some other encrypted protocol.
 Telnet is an outdated form of communication that does not use authentication to access a shell. [CWE-311](https://cwe.mitre.org/data/definitions/311.html) explains the weakness being; the software does not encrypt sensitive or critical information before storage or transmission. The lack of encryption could mean breaches of confidentiality, integrity, and accountability. The file [device_tracker.py](https://github.com/home-assistant/core/blob/dev/homeassistant/components/actiontec/device_tracker.py) uses telnet to get data for an Actiontec MI424WR (Verizon FIOS) router starting on line 93. After further research, the Actiontec MI424WR (Verizon FIOS) routers are outdated and do not support SSH. The only [work around](https://forums.verizon.com/t5/Fios-Internet/Remotely-restart-FIOS-MI424WR-Router/td-p/546593) found for safe remote access is to use an SSH device on the network to access the router. However, this solution is not feasible for Home Assistant, and the router is so outdated, there are likely little to none in use. For these reasons, so we decided not to submit a pull request. 
 
-### B402	A FTP-related module is being imported.  FTP is considered insecure. Use SSH/SFTP/SCP or some other encrypted protocol.
+### B321	A FTP-related module is being called. FTP is considered insecure. Use SSH/SFTP/SCP or some other encrypted protocol.
```

Comment:
idk if its Github or what but make sure to say that this is for B321 and B402 
### B321 & B402